server/db/companies.js

```diff
@@ -24,3 +26,12 @@ function addCompany (company, db = knex) {
     .insert(company)
     .then(companyId => companyId[0])
 }
+
+function getProfile(id, db = knex) {
+  return db('roles').where(id = 'Company_ID')
+}
+
+function addProfile(profile, db = knex) {
+  return db('role_counts')
+    .insert(profile)
```

Comment:
The properties on this `profile` object will be in camelCase. You will need to create an "equivalent" object that has train_case properties so it can be inserted.